mlir/lib/Conversion/QCToQIR/QCToQIR.cpp

```
@@ -1036,6 +1138,44 @@ struct ConvertQCYieldOp final : StatefulOpConversionPattern<YieldOp> {
+  patterns.add<ConvertQCStaticOp>(typeConverter, ctx, &state);
+  patterns.add<ConvertQCMeasureOp>(typeConverter, ctx, &state);
+  patterns.add<ConvertQCResetOp>(typeConverter, ctx, &state);
+  patterns.add<ConvertQCGPhaseOp>(typeConverter, ctx, &state);
+
+  // Note: `MQT_GATE_TABLE` is defined in `mlir/Conversion/GateTable.h`.
+#define MQT_ADD_QC_TO_QIR_UNITARY(                                             \
+    KEY, TARGETS, PARAMS, QCO_OP, QC_OP, JEFF_KIND, JEFF_OP,                   \
+    JEFF_BASE_ADJOINT, JEFF_CUSTOM_NAME, JEFF_PPR, QIR_KIND, QIR_FN)           \
+  patterns.add<ConvertQCUnitaryOpQIR<QC_OP, (TARGETS), (PARAMS), &(QIR_FN)>>(  \
+      typeConverter, ctx, &state);
+  MQT_GATE_TABLE(MQT_ADD_QC_TO_QIR_UNITARY)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If MQT_GATE_TABLE is a macro then please configure it.

(unknownMacro)